roles/ycsb/tasks/main.yml

```diff
@@ -0,0 +1,208 @@
+---
+- name: Get benchmark state
+  k8s_facts:
+    api_version: ripsaw.cloudbulldozer.io/v1alpha1
+    kind: Benchmark
+    name: "{{ meta.name }}"
+    namespace: "{{ operator_namespace }}"
+  register: resource_state
+
+- k8s_status:
+    api_version: ripsaw.cloudbulldozer.io/v1alpha1
+    kind: Benchmark
+    name: "{{ meta.name }}"
+    namespace: "{{ operator_namespace }}"
+    status:
+      state: Building
+      complete: false
+  when: resource_state.resources[0].status.state is not defined
+
+- name: Get benchmark state
+  k8s_facts:
+    api_version: ripsaw.cloudbulldozer.io/v1alpha1
+    kind: Benchmark
+    name: "{{ meta.name }}"
+    namespace: "{{ operator_namespace }}"
+  register: resource_state
+
+- block:
+
+  - name: setting ycsb_workload var
+    set_fact:
+      ycsb_workload: "workloada"
```

Comment:
I suggest that we move away from using the variable ycsb_workload in the load phase, as we're overwriting this variable again in when state is "Running Load" and setting it to the same value workloada, and finally in run phase we're iterating over various workloads.

Reply:
That's fair. I updated the load side variable to be ycsb_workload_load